src/components/ErrorPage.vue

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
```

Comment:
I'd suggest making this look exactly like the success page, but with red/amber text and a different image. Just use a placehold.it or something or grab an image from google